rttest/test/test_api.cpp

```diff
@@ -167,3 +167,21 @@ TEST(TestApi, running) {
   EXPECT_EQ(0, rttest_finish());
   EXPECT_EQ(0, rttest_running());
 }
+
+TEST(TestApi, timespec_to_uint64) {
+  // failed values in 32bit OS (#94)
+  time_t sec = 363464;
+  long nsec = 232837182; // NOLINT for C type long
+  uint64_t v = 363464232837182;
+
+  struct timespec t;
+  t.tv_sec = sec;
+  t.tv_nsec = nsec;
```

Comment:
Small nit, but you can simplify this a bit:

```
  struct timespec t;
  t.tv_sec = 363464;
  t.tv_nsec = 232837182;
  uint64_t v = 363464232837182;
```

(and then do your `EXPECT_EQ` against t.tv_sec and t.tv_nsec below).

Reply:
Thank you for comment.
I fixed at b33dce199590ca487de3c8d3c8736c343f7c37f5.